src/screenComponents/radarView.cpp

```diff
@@ -756,23 +776,10 @@ void GuiRadarView::drawObjects(sf::RenderTarget& window)
         }
     };
 
-    // First draw all objects that are maybe hidden.
-    window.setActive(true);
-    glStencilFunc(GL_EQUAL, as_mask(RadarStencil::InBoundsAndVisible), as_mask(RadarStencil::InBoundsAndVisible));
-    for (auto obj : visible_objects)
-    {
-        if (obj->canHideInNebula())
-        {
-            draw_object(obj);
-        }
-    }
-
-    // Second, draw all objects that can't hide.
```

Comment:
This bit was interesting: the previous iteration had a complex path of using the 'alpha window' or the 'background' one depending on whether the object could hide in a nebula or not.

My initial understanding was that I had to honor that, but from a closer look, it appears the filtering is already done before (correctly handling friendlies etc) so... Looks like we can get away by just drawing the list of visible objects as-is.